src/chat/components/InputToolbar.tsx

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import {
+  View,
+  StyleSheet,
+  ScrollView,
+  TouchableOpacity,
+  Image,
+  StyleProp,
+  ViewStyle,
+  ImageStyle,
+} from 'react-native';
+import {
+  Composer,
+  InputToolbarProps,
+  SendProps,
+} from 'react-native-gifted-chat';
+
+interface IInputToolbar extends InputToolbarProps<any>, SendProps<any> {
+  isShowCamera?: boolean;
+  isShowPhotoGallery?: boolean;
+  togglePhotoGallery?: () => void;
+  containerStyle?: StyleProp<ViewStyle>;
+  composeWrapperStyle?: StyleProp<ViewStyle>;
+  composerTextInputStyle?: StyleProp<ViewStyle>;
+  customViewStyle?: StyleProp<ViewStyle>;
+  iconCamera?: any;
+  iconGallery?: any;
+  iconSend?: any;
+  iconStyle?: StyleProp<ImageStyle>;
+  iconSize?: number;
+  iconMargin?: number;
+}
+
+export const PressableIcon: React.FC<{
```

Comment:
move this implement to `PressableIcon.tsx`

Reply:
Yes, I moved,  please check it